lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1191,36 +1207,42 @@ mkUnsignedTx era ttl cs md wdrls certs fees =
         ShelleyBasedEraShelley -> toShelleyTxOut
         ShelleyBasedEraAllegra -> toAllegraTxOut
         ShelleyBasedEraMary -> toMaryTxOut
+        ShelleyBasedEraAlonzo -> error "todo: toAlonzoTxOut"
```

Comment:
```suggestion
        ShelleyBasedEraAlonzo -> error "todo(ADP-952): toAlonzoTxOut"
```

Reply:
If you use something like this:
```haskell
error "toAlonzoTxOut unimplemented" -- TODO: [ADP-952] toAlonzoTxOut
```
then it's easy to find all your TODOs with lentil:
```shell
$ lentil -t ADP-952 lib
263 source files [****************] 100%
lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs
   521  fromAlonzoBlock [ADP-952]
   923  fromAlonzoTx [ADP-952]
  1148  datumHash [ADP-952]
  1167  datumHash [ADP-952]
  1186  datumHash [ADP-952]
 
lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs
  1173  We presumably need to provide the protocol params if our tx uses
        scripts? [ADP-952]
  1220  toAlonzoTxOut [ADP-952]
```